tst/EnergyPlus/unit/SimulationManager.unit.cc

```diff
@@ -112,6 +112,8 @@ TEST_F(EnergyPlusFixture, Test_PerformancePrecisionTradeoffs_DirectSolution_Mess
 
     EXPECT_TRUE(process_idf(idf_objects, false));
 
+    SimulationManager::GetProjectData(*state);
+
```

Comment:
Another revert, unit tests are now responsible for setting up the inputs. Either this way or calling init_state directly.